src/docs/releases/1.9.0.md

```diff
@@ -204,6 +204,9 @@ In the past, we utilized the injection of `ISmsProvider` for sending SMS message
 
 Additionally, `Twilio` provider is no longer enabled by default. If you want to use Twilio SMS provider, you must enable the provider by visiting the email settings `Configuration` > `Settings` > `Email` and see the `Twilio` tab.
 
+### `SectionDisplayDriver.UpdateAsync()` now receives an `UpdateEditorContext` parameter, not `BuildEditorContext`
+
+`SectionDisplayDriver` now uses the appropriate UpdateEditorContext on its Update methods. Every driver that inherits from it will be required to update their contexts. See [the corresponding pull request](https://github.com/OrchardCMS/OrchardCore/pull/15488) for context and for examples of the changes you have to implement in your own modules.
```

Comment:
```suggestion
### Display Management

In this release, the signatures of the `UpdateAsync()` method within the `SectionDisplayDriver` base class have undergone modifications. Previously, these signatures accepted the `BuildEditorContext` parameter. However, with this update, all signatures now require the `UpdateEditorContext` instead. This alteration necessitates that every driver inheriting from this class adjusts their contexts accordingly.

Here are the updated signatures:

1. **From:**
   ```csharp
   Task<IDisplayResult> UpdateAsync(TModel model, TSection section, IUpdateModel updater, BuildEditorContext context)
   ```
   **To:**
   ```csharp
   Task<IDisplayResult> UpdateAsync(TModel model, TSection section, IUpdateModel updater, UpdateEditorContext context)
   ```

2. **From:**
   ```csharp
   Task<IDisplayResult> UpdateAsync(TSection section, IUpdateModel updater, BuildEditorContext context)
   ```
   **To:**
   ```csharp
   Task<IDisplayResult> UpdateAsync(TSection section, IUpdateModel updater, UpdateEditorContext context)
   ```

3. **From:**
   ```csharp
   Task<IDisplayResult> UpdateAsync(TSection section, BuildEditorContext context)
   ```
   **To:**
   ```csharp
   Task<IDisplayResult> UpdateAsync(TSection section, UpdateEditorContext context)
   ```

These adjustments ensure compatibility and adherence to the latest conventions within the `SectionDisplayDriver` class.
```

Reply:
At this point commit the suggestions you would like Mike. Or push a commit directly.